examples/aws-sso/Earthfile

```
@@ -0,0 +1,85 @@
+VERSION 0.7
+FROM amazon/aws-cli
+ARG --global EARTHLY_CI
```

should this default to `false` rather than the empty string?
```suggestion
ARG --global EARTHLY_CI=false
```

---

I've had no problem with this being empty, but I have no problem setting it to false if you think that's better.